dashboard/app/assets/stylesheets/application.scss

```
@@ -1927,6 +1927,23 @@ a.download-video {
   display: flex;
   flex-wrap: wrap;
 
+  .required {
+    font-weight: 600;
+
+    &.not-bold {
+      font-weight: normal;
+    }
+
+    & > * {
+      font-weight: 600;
+    }
+
+    &::after {
+      content: "*";
+      display: inline;
+    }
+  }
+
```

Try this to clean up the css a bit!

```suggestion
  .required, 
  .required > * {
    font-weight: 600;

    &.not-bold {
      font-weight: normal;
    }

    &::after {
      content: "*";
      display: inline;
    }
  }

```